pkg/pillar/zedUpload/commonutils/types.go

```diff
@@ -0,0 +1,33 @@
+// Copyright(c) 2017-2018 Zededa, Inc.
+// All rights reserved.
```

Comment:
Use the same line as in the other files which references the license

Reply:
sure

Reply:
Any update?

Reply:
I referred from the other files i.e., datastore_req.go, datastore_aws.go.
looks like they do have the same format.

Reply:
OK so we'll have to fix this separately